src/executor/evaluate/mod.rs

```diff
@@ -197,6 +197,88 @@ pub async fn evaluate<'a, T: 'static + Debug>(
         Expr::Wildcard | Expr::QualifiedWildcard(_) => {
             Err(EvaluateError::UnreachableWildcardExpr.into())
         }
+        Expr::Case {
+            operand,
+            when_then,
+            else_result,
+        } => {
+            use super::EvaluateError;
+            use itertools::Itertools;
+            use std::mem::discriminant as disc;
```

Comment:
Let's merge this into the top `use`.
If top `use` is too verbose, then that would be the proper timing for splitting file.
However, let's simply merge it for now.

Reply:
Alrighty ✅